specs/DesignDiscussions/InputLabels.md

```diff
@@ -413,8 +376,84 @@ This document has primarily focused on `Input.Text` in it's examples, but the la
 ```
 ![img](assets/InputLabels/Input.Toggle.PNG)
 
+## Feature Cost estimation
+
+This feature focuses on adding a text element to an input element which will have to be implemented in all of our renderers and the designer. It's important to note that this feature covers the accessibility aspect which should be reviewed by people who have expertise in this aspect.
+
+For the development of this feature the following costs have been estimated for all platforms like this:
+
+| Task | Cost (days) | Previous requirements | Includes |
+| --- | --- | --- | --- |
+| C++ Object Model | 4 | Spec completion | Tests |
+| UWP Implementation | 4 | C++ Object Model | Tests and override sample |
+| Android Implementation | 4 | C++ Object Model | Tests and override sample | 
+| iOS Implementation | 4 | C++ Object Model | Override sample |
+| C# Object Model | 4 | Spec completion | Tests |
+| .NET Implemenation | 4 | C# Object Model | Override sample |
+| HTML Implementation | 4 | C# Object Model | Tests | 
+| JavaScript Implementation | 14 | Spec completion | Changes to designer |
+| Documentation | 6 | Spec completion, UWP, Android, iOS and .NET implementations | Changes to schema explorer and guidance on how to override inputs | 
```

Comment:
Bug Bash ? Probably add in a couple of days worth here to account for prep + bash